tests/unit-tests/test_gnn.py

```diff
@@ -748,24 +759,54 @@ def test_gat_node_prediction(device):
     th.distributed.destroy_process_group()
     dgl.distributed.kvstore.close_kvstore()
 
-def create_rgcn_edge_model(g, num_ffn_layers):
+def create_rgcn_edge_model(g, num_ffn_layers, edge_feat_name=None):
     model = GSgnnEdgeModel(alpha_l2norm=0)
 
-    feat_size = get_node_feat_size(g, 'feat')
-    encoder = GSNodeEncoderInputLayer(g, feat_size, 4,
+    nfeat_size = get_node_feat_size(g, 'feat')
+    node_encoder = GSNodeEncoderInputLayer(g, nfeat_size, 4,
                                       dropout=0,
                                       use_node_embeddings=True)
-    model.set_node_input_encoder(encoder)
+    efeat_size = get_edge_feat_size(g, 'feat')
+    edge_encoder = GSEdgeEncoderInputLayer(g, efeat_size, 4, dropout=0)
+    model.set_node_input_encoder(node_encoder)
+    model.set_edge_input_encoder(edge_encoder)
 
     gnn_encoder = RelationalGCNEncoder(g, 4, 4,
                                        num_bases=2,
                                        num_hidden_layers=1,
+                                       edge_feat_name=edge_feat_name,
                                        dropout=0,
                                        use_self_loop=True)
     model.set_gnn_encoder(gnn_encoder)
     model.set_decoder(MLPEdgeDecoder(model.gnn_encoder.out_dims,
-                                     3, multilabel=False, target_etype=("n0", "r1", "n1"),
+                                     10, multilabel=False, target_etype=("n0", "r1", "n1"),
```

Comment:
Why change this?

Reply:
Because the test data's label is 10 classes, not 3. The previous test code is wrong, but not be used in test cases.

Below is the test data code in `data_utils.py`.

```
    hetero_graph.nodes['n1'].data['label'] = th.randint(10, (hetero_graph.number_of_nodes('n1'), ))

    ......

    hetero_graph.edges['r1'].data['label'] = th.randint(10, (hetero_graph.number_of_edges('r1'), ))

```